pom.xml

```diff
@@ -188,7 +188,7 @@
                         <configuration>
                             <referenceVersion>2.23ea0</referenceVersion>
                             <artifactsURI>https://teamcity.chronicle.software/repository/download</artifactsURI>
-                            <binaryCompatibilityPercentageRequired>96</binaryCompatibilityPercentageRequired>
+                            <binaryCompatibilityPercentageRequired>95</binaryCompatibilityPercentageRequired>
```

Comment:
it's not good to change this.

Reply:
This comes from making some LongConverters use inheritance.